members/misha.md

```diff
@@ -1,38 +1,32 @@
-# Hello there! 👋
+# Rugira prince
```

Comment:
hey! just to recheck - seems like Prince's name is in your file?

Reply:
you're great, I was checking md prince and just forgot to return my md, well noted